west.yml

```diff
@@ -332,10 +333,11 @@ manifest:
       groups:
         - tee
     - name: uoscore-uedhoc
-      revision: 5fe2cb613bd7e4590bd1b00c2adf181ac0229379
+      revision: pull/61/head
+      url: https://github.com/eriptic/uoscore-uedhoc
       path: modules/lib/uoscore-uedhoc
     - name: zcbor
-      revision: 67fd8bb88d3136738661fa8bb5f9989103f4599e
+      revision: 0.8.0
```

Comment:
Need to use a commit ID for this, not a tag

Reply:
Done